libs/elements/src/select/vl-select.element.ts

```diff
@@ -67,6 +69,17 @@ export class VlSelect extends vlFormValidationElement(BaseElementOfType(HTMLSele
         }
     }
 
+    /**
+     * @param name {String}
+     * @param oldValue {String} - wordt niet gebruikt, maar nodig voor de signature super.__changeAttribute
+     * @param newValue {String}
+     */
+    attributeChangedCallback(name: string, oldValue: string, newValue: string) {
```

Comment:
Is dit nodig? Normaal gezien als disabled veranderd komt die binnen in de `_disabledChangedCallback` methode.
Deze methode overriden lijkt me ook gevaarlijk, zeker aangezien je de super.attributeChangedCallback niet aanroept.

Reply:
Ja dit is nodig, als ik het verwijder dan heb ik hetzelfde probleem als voorheen, ik krijg het "look en feel" van een 'disabled' input maar ik kan nog steeds op de select klikken en de opties zien.

Ik moet luisteren naar de wijzigingen van dit attribuut en het 'disabled'-attribuut in de input injecteren. Dit is de makkelijkste en snelste manier die ik heb gevonden zonder al te veel in detail te treden waarom de `disable()` van DV zich niet gedraagt zoals verwacht..

Reply:
terechte opmerking, de component brak, opgelost

attributeChangedCallback(name: string, oldValue: string, newValue: string) {
        if (name === 'data-vl-disabled') {
            this._disabledChangedCallback(newValue);
        } else {
            super.attributeChangedCallback(name, oldValue, newValue);
        }
    }